.github/workflows/test.yml

```diff
@@ -0,0 +1,40 @@
+# This is a simple workflow that runs the publisher and copies the output to https://<owner>.github.io/<repo>/index.html
+# Based on the instructions from Elliot Silver, available from: https://www.argentixinfo.com/archives/156
+
+# Make sure your repo has a branch called gh-pages
+
+name: Test compilation
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - name: Update the image to the latest publisher
+        uses: docker://hl7fhir/ig-publisher-base:latest
+        with:
+          # Get the latest publisher - don't run the batch script but run the line directly
+          args: curl -L https://github.com/HL7/fhir-ig-publisher/releases/latest/download/publisher.jar -o ./input-cache/publisher.jar --create-dirs
+
+      - name: Run the IG publisher
+        uses: docker://hl7fhir/ig-publisher-base:latest
+        with:
+          # Run the publisher - don't run the batch script but run the line directly
+          args: java -jar ./input-cache/publisher.jar publisher -ig .
```

Comment:
so this simply just build the IG without publishing it anywhere, how would we visualize the changes pushed on given branch ??

Reply:
I think as i comented in slack , 
we can still build the changes on a given branch and still deploy them to gh-pages  , in order to vusalise the changes in the IG ,
but then re-run the Actions on the main branch to Update the IG to the true version

Reply:
I think it would make sense to keep the IG build on PRs, since it is a CI version of the IG.

Reply:
However, we should probably also have a more official location for hosting a release of the IG

Reply:
So then, only releases would trigger a publication to that location, but we could keep the original full build on PRs

Reply:
Yes that makes sense